### ISSUE TITLE
Suggestion - enable example browsing and copying from csolution command

### ISSUE THREAD

**Issue by markgsaunders** [Open-CMSIS-Pack/cmsis-toolbox#146]
From the command line (bash), CMSIS does not have a simple means of creating applications from a vendor's set of examples. In uVision and Keil Studio the installed packs are scoured for examples and the user can browse them, choose one, and ask the tool to create a project for it. On the command line, however, the user has to find the examples and manually copy the contents from a github repo, the CMSIS_PACK_ROOT folder, or elsewhere. It's not the most robust situation because it entails manually pulling parts from the vendor's folder structure, and presupposes some user knowledge of where to look.
I think the csolution program, which already has commands like list packs and list boards, could be extended to list examples. This would at least tell the user where to find "blinky".
Adding to that, it would be fantastic if csolution (or some other program) could be used to make a copy of the example by specifying the source pack, version, and example. As a user I could then use csolution to browse for examples (csolution list examples) and then reliably create a copy of one in seconds (csolution Vendor::DFP@1.0.0 blinky).

**Comment by markgsaunders** [comment]
Yes, this is an enhancement or usability issue rather than something that cannot be solved by the user any other way. But I think it would be a valuable addition!

**Comment by ReinhardKeil** [comment]
We believe that this is handled better in the VSCode extension where examples are already listed 
https://github.com/ARM-software/vscode-cmsis-csolution
